pages/common/cryfs.md

```diff
@@ -0,0 +1,20 @@
+# cryfs
+
+> A cryptographic filesystem for the cloud.
+> More information: <https://www.cryfs.org/>.
+
+- Mount an encrypted file system. If this is the first time, start the initialization wizard:
```

Comment:
Perhaps it is better to say - "The initialization wizard will be started on the first time."

Reply:
I've fix this.